azure-pipelines.yml

```diff
@@ -41,4 +41,5 @@ jobs :
       displayName: Windows
       vmImage: 'vs2017-win2016'
       options: config/azure/options.windows
+      envSetupCmd: '"C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64'
```

Comment:
Does it work if this becomes:
```
    envSetupCmd: 'call "C:\Prog...
```
And then in `build.yaml`:
```
parameters:
    envSetupCmd: 'g++ --version'
```
``` 
- script: |
    ${{ parameters.envSetupCmd }}
    scons -j ${{ parameters.threads }} install ...
```

then we don't need two different windows/posix tasks?

Reply:
Great catch, that's definitely an improvement. It's included in c51d871

Reply:
Awesome, thanks so much. Looks great now!   

Now I just need to get my `imageEnv` issues sorted out.  Ill look into the test failures too and sort a merge out once I've resolved those two issues.

Nothing else to be done your end I don't think. Thank you - really appreciate all the help getting this together!

Reply:
You're very welcome! Great to see things one step closer to The Convergence :)